instrumentation/runtime-telemetry/runtime-telemetry-java8/javaagent/src/main/java/io/opentelemetry/instrumentation/javaagent/runtimemetrics/java8/JarAnalyzer.java

```diff
@@ -172,25 +199,26 @@ public void run() {
         try {
           // TODO(jack-berg): add ability to optionally re-process urls periodically to re-emit
           // events
-          processUrl(eventEmitter, archiveUrl);
+          processUrl(eventEmitter, archiveUrl, debug);
         } catch (Throwable e) {
-          logger.log(Level.WARNING, "Unexpected error processing archive URL: " + archiveUrl, e);
+          logger.log(
+              debug ? WARNING : FINE, "Unexpected error processing archive URL: " + archiveUrl, e);
         }
       }
-      logger.warning("JarAnalyzer stopped");
+      logger.log(debug ? WARNING : FINE, "JarAnalyzer stopped");
     }
   }
 
   /**
    * Process the {@code archiveUrl}, extracting metadata from it and emitting an event with the
    * content.
    */
-  static void processUrl(EventEmitter eventEmitter, URL archiveUrl) {
+  static void processUrl(EventEmitter eventEmitter, URL archiveUrl, boolean debug) {
     JarDetails jarDetails;
     try {
       jarDetails = JarDetails.forUrl(archiveUrl);
     } catch (IOException e) {
-      logger.log(Level.WARNING, "Error reading package for archive URL: " + archiveUrl, e);
+      logger.log(debug ? WARNING : FINE, "Error reading package for archive URL: " + archiveUrl, e);
```

Comment:
This reads backwards to me. When I enable debug, I expect _more_ verbosity and more granular messages.

Reply:
it's a bit confusing. the reason we do this kind of thing for muzzle is that we want the muzzle warnings to jump out (by being labelled as `WARNING`), but at the same time, we only want to emit them when the java agent debug flag is enabled, since they are somewhat expected and we don't imbue panic into users

muzzle is a (reasonably) special case that we're aware of

@laurit do you think the JarAnalyzer warnings are important enough to get the same treatment? otherwise we could just do the simple thing and emit as `FINE`, they will still have stack traces which will make them very visible at that level

Reply:
@breedx-splk the main issue is that `INFO` and above are sent to stderr. We have to be careful with what we send there, most users don't want to see our stack traces there.

@trask hopefully we'll eventually reach a state where our logs can be used to debug customer issues, then having abnormal situations logged at `WARN` and `ERROR` would be really nice. An alternative would be to have separate logger for logging to console output. Or we could set it up so that only things that we explicitly want in the output like the informative line from `VersionLogger` at the agent start and exporter errors  go there (or perhaps send info+ logs from sdk to console) and the rest goes to log file or is visible only when agent debug is enabled regardless of the log level.

Reply:
thx, what about leaving these logs at `WARN` now that you've fixed the Payara issue? (btw, I don't really have strong opinions on any of this, happy with whatever you recommend)

Reply:
I changed the logging back to as it was, if needed jar analyzer can be disabled separately.